content/utilities/mini-map/plugin.js

```diff
@@ -175,6 +177,8 @@ class Plugin {
             ctx.fillStyle = '#24247d'; // outer nebula
           } else if (data[i].type === 2) {
             ctx.fillStyle = '#000000'; // deep space
+          } else if (data[i].type === 3) {
+            ctx.fillStyle = "magenta"; // Corrupted... Will sub out with actual color
```

Comment:
I mined a chunk and color-pickered it
```suggestion
            ctx.fillStyle = "#250025"; // Corrupted
```